lib/mutant/integration/minitest.rb

```diff
@@ -0,0 +1,155 @@
+require 'minitest'
+
+# monkey patch for minitest
+module Minitest
+  # Prevent autorun from running tests when the VM closes
+  #
+  # Mutant needs control about the exit status of the VM and
+  # the moment of test execution
+  #
+  # @api private
+  #
+  # @return [nil]
+  def self.autorun
+  end
+
+end # Minitest
+
+module Mutant
+  class Integration
+    # Minitest integration
+    class Minitest < self
+      TEST_METHOD_PREFIX = 'test_'.freeze
+      TEST_FILE_PATTERN  = './test/**/{test_*,*_test}.rb'.freeze
```

Comment:
No problem. I stopped looking for deviations since it sounds like you agree this needs to be configurable. As one last check I grepped out the `gem.test_files` setting in each gemspec. Here are the unique entries (minus a few weirdos):

```ruby
Dir.glob("spec/**/*")
Dir["spec/**/*.rb"]
Dir["test/spec_*.rb"]
Dir["test/test_*.rb"]
`git ls-files -- Appraisals {spec}/*`.split("\n")
`git ls-files -- spec/*`.split("\n")
`git ls-files -- test/*`.split("\n")
`git ls-files -- test`.split($/)
`git ls-files -- {spec}/*`.split("\n")
`git ls-files -- {test,spec,features}/*`.split("\n")
`git ls-files -- {test}/*`.split("\n")
`git ls-files spec/{unit,integration}`.split($/)
gem.files.grep(%r{^(test|spec|features)/.*\.rb})
gem.files.grep(%r{^(test|spec|features)/})
gem.files.grep(%r{^(test|spec|features)/}).grep(%r{/test_[^/]+\.rb$})
gem.files.grep(%r{^spec/})
s.files.select { |path| path =~ /^[spec|tests]\/.*_[spec|tests]\.rb/ }
s.files.select {|path| path =~ /^test\/.*_test.rb/}
spec.files.grep(%r{^(test|spec|features)/})
spec.files.grep(%r{^(test|spec|features|acceptance)/})
spec.files.grep(%r{^(test|spec|features|examples)/})
spec.files.grep(%r{^test/})
```